piwik-sdk/src/main/java/org/piwik/sdk/TrackMe.java

```diff
@@ -20,7 +19,13 @@
 public class TrackMe {
     private static final int DEFAULT_QUERY_CAPACITY = 14;
     private final HashMap<String, String> mQueryParams = new HashMap<>(DEFAULT_QUERY_CAPACITY);
-    private final CustomVariables mScreenCustomVariable = new CustomVariables();
```

Comment:
:grin: I'd still love to get rid of "visit custom variables" in the `Tracker`, but this is a good start.